packages/common/src/core/HubItemEntity.ts

```diff
@@ -256,6 +256,25 @@ export abstract class HubItemEntity<T extends IHubItemEntity>
     });
   }
 
+  /**
+   * Sets whether or not the followers group is discussable
+   * @param isDiscussable
+   */
+  async setFollowersGroupIsDiscussable(isDiscussable: boolean) {
+    const group = await this.getFollowersGroup();
+    const typeKeywords = setDiscussableKeyword(
+      group.typeKeywords,
+      isDiscussable
+    );
+    const afterGroup = await updateGroup({
```

Comment:
Minor, const is never used, maybe just `await updateGroup({ .... })`?

Also curious if we should maybe wrap this in a try/catch and throw a more helpful error like `Failed to set discussability of followers group ${this.entity.followersGroupId} to ${isDiscussable}`.